### PR TITLE
Sabas merge

### PR DESCRIPTION
**Pull request by ArcaneDots**
Site seems to be working.  Translations should be complete but are rough. Plan to reduce number of translation keys used once this is PR is merged.

### CODE REVIEW

**Comment by ArcaneDots** [comment]
Fully merged improvements from the Sabas fork back into main project. Fixed a few bugs and verified that notes were generated correctly.

**Comment by ArcaneDots** [comment]
The Sabas fork already had fixes for issues #46 and #47

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-onosm.org
+su.openstreetmap.it
```
Comment (iandees) [inline CNAME]: Please don't change the CNAME.

**Review by iandees — CHANGES_REQUESTED** [review]
Thanks for taking the time to do this.

Can you please fix a couple things for me:
* The CNAME, LICENSE, and README.md files should not be marked executable
* The CNAME file should stay at onosm.org otherwise the site will stop working

File: CNAME (hunk 1 above)
Comment (ArcaneDots) [reply CNAME]: Must have missed these during the merge

**Review by iandees — APPROVED** [review]
Thanks!

**Pull request merged**